easy-ui-react/src/TextField/TextField.stories.tsx

```diff
@@ -49,6 +49,17 @@ export const Icon: Story = {
   },
 };
 
+export const PrefixAndSuffix: Story = {
+  render: Template.bind({}),
+  args: {
+    label: "Label",
+    placeholder: "Placeholder text",
+    helperText: "Optional helper text",
+    textAtStart: "$",
+    textAtEnd: "2.50",
```

Comment:
probably best to use a smaller example like "in" so that it fits within the 24px by 24px paradigm